### PR TITLE
improved student registration form to reflect fields that are relevant…

### PR DESCRIPTION
**Pull request by majakudlicka**
… and fixed minor bugs

relates to #23

### CODE REVIEW

File: src/components/Form_Register_Student.js [hunk 1]
```diff
@@ -10,68 +12,94 @@ class Form_Register_Student extends Component {
...
+          <input className="form__field-input" id="Phone number" type="text" placeholder="Phone number"  />
 
         </div>
 
         <div className="form__field-wrapper">
-        <label className="form__field-label" htmlFor="Phone number">Phone number</label>
-          <input className="form__field-input" id="Phone number" type="Phone number" value={this.props.data.password}
-           placeholder="Phone number"  onChange={this._changePassword.bind(this)} />
+        <label className="form__field-label" htmlFor="Job categories">Pick up to 8 jobs categories</label>
+        <input className="form__field-input" id="job category1" type="text" placeholder="Select" list="jobs"  />
```
Comment (des-des) [inline src/components/Form_Register_Student.js]: are these meant to be `type="text"`?

File: src/components/Form_Register_Student.js [hunk 2]
```diff
@@ -10,68 +12,94 @@ class Form_Register_Student extends Component {
...
+        <input className="form__field-input" id="job category2" type="text" placeholder="Select" list="jobs"  />
+        <input className="form__field-input" id="job category3" type="text" placeholder="Select" list="jobs"  />
+        <input className="form__field-input" id="job category4" type="text" placeholder="Select" list="jobs"  />
+        <input className="form__field-input" id="job category5" type="text" placeholder="Select" list="jobs"  />
+        <input className="form__field-input" id="job category6" type="text" placeholder="Select" list="jobs"  />
+        <input className="form__field-input" id="job category7" type="text" placeholder="Select" list="jobs"  />
+        <input className="form__field-input" id="job category8" type="text" placeholder="Select" list="jobs"  />
+        <datalist id="jobs">
+        <option value="Dog-walking" />
```
Comment (des-des) [inline src/components/Form_Register_Student.js]: indentation

File: src/components/Form_Register_Student.js [hunk 3]
```diff
@@ -10,68 +12,94 @@ class Form_Register_Student extends Component {
...
+        <label className="form__field-label" htmlFor="Phone number">Phone number</label>
+          <input className="form__field-input" id="Phone number" type="text" placeholder="Phone number"  />
 
         </div>
 
         <div className="form__field-wrapper">
-        <label className="form__field-label" htmlFor="Phone number">Phone number</label>
-          <input className="form__field-input" id="Phone number" type="Phone number" value={this.props.data.password}
-           placeholder="Phone number"  onChange={this._changePassword.bind(this)} />
+        <label className="form__field-label" htmlFor="Job categories">Pick up to 8 jobs categories</label>
```
Comment (des-des) [inline src/components/Form_Register_Student.js]: indentation

File: src/components/Form_Register_Student.js [hunk 4]
```diff
@@ -10,68 +12,94 @@ class Form_Register_Student extends Component {
...
 
         </div>
 
+
         <div className="form__field-wrapper">
-        <label className="form__field-label" htmlFor="Job categories">Job categories</label>
-          <input className="form__field-input" id="Job categories" type="Job categories" value={this.props.data.password}
-           placeholder="Job categories"  onChange={this._changePassword.bind(this)} />
+        <label className="form__field-label" htmlFor="Bio">Bio</label>
+          <input className="form__field-input" id="Bio" type="text" placeholder="Bio"  />
```
Comment (des-des) [inline src/components/Form_Register_Student.js]: indentation

File: src/components/Form_Register_Student.js [hunk 5]
```diff
@@ -10,68 +12,94 @@ class Form_Register_Student extends Component {
...
+           autoCapitalize="off"
+           spellCheck="false" />
 
         </div>
         <div className="form__field-wrapper">
         <label className="form__field-label" htmlFor="password">Password</label>
-          <input className="form__field-input" id="password" type="password" value={this.props.data.password}
-           placeholder="password"  onChange={this._changePassword.bind(this)} />
+          <input className="form__field-input"
+          id="password" type="password"
```
Comment (des-des) [inline src/components/Form_Register_Student.js]: intentation

File: src/components/Form_Register_Student.js (hunk 1 above)
Comment (des-des) [inline src/components/Form_Register_Student.js]: seems more like a checkbox like thing

**Comment by majakudlicka** [comment]
I have made the changes but since this code is already part of the other pull request, I am closing this one (also I cant make this brach to work anymore after we made all the changes to webpack server etc)

**Pull request closed without merging**